runtime/common/bigint.go

```diff
@@ -54,5 +56,9 @@ func OverEstimateBigIntFromString(s string) int {
 	// First convert to word, and then find the size,
 	// to be consistent with `common.BigIntByteLength`
 	wordLen := (bitLen + 63) / 64
-	return (wordLen + 4) * BigIntWordSize
+	result := wordLen * BigIntWordSize
+	if result < minBigIntLength {
+		return minBigIntLength
+	}
+	return result
```

Comment:
Now that I've been reminded inputs are base 10 and not base 16 :)  I think we can be simplified to something like:
```
        if wordLen == 1 {
		return BigIntWordSize
	}
	return (wordLen + 4) * BigIntWordSize
```

Reply:
Same question as above. How can this be changed to match the different bases (given that it now supports bases 2, 8, 10, and 16)?

Reply:
It's same as the code I wrote above. If `wordLen` is `1`, then `BigIntWordSize` are enough memory, otherwise it is `(wordLen + 4) * BigIntWordSize`. 
I don't think the constant `minBigIntLength` is needed 🤔